packages/frontend-2/components/viewer/explorer/Filters.vue

```diff
@@ -246,4 +230,25 @@ const refreshColorsIfSetOrActiveFilterIsNumeric = () => {
   // removePropertyFilter()
   applyPropertyFilter()
 }
+
+const isRevitProperty = (key: string): boolean => {
+  return key.startsWith('parameters.') && key.endsWith('.value')
+}
+
+const getPropertyName = (key: string): string => {
+  if (!key) return 'Loading'
+
+  if (key === 'level.name') return 'Level Name'
+  if (key === 'speckle_type') return 'Object Type'
+
+  if (isRevitProperty(key)) {
+    const correspondingName = props.filters.find(
```

Comment:
Are we sure that this is gonna be a StringPropertyInfo? PropertyInfo can inherit both StringPropertyInfo and NumericPropertyInfo filters.

Better to define and use a type guard instead: `isStringPropInfo(i: PropertyInfo): i is StringPropertyInfo => i.type === 'string'`